### PR TITLE
Re-enable tests using host ports in CRI builds

### PR DESCRIPTION
**Pull request by yujuhong**
The docker-CRI implementation now supports the host port feature.

### CODE REVIEW

**Comment by yujuhong** [comment]
These suites are not submit-queue blocking, so it should be safe to re-enable the tests. If the tests pass consistently, I'll upload PRs to make them submit queue blocking.

**Comment by yujuhong** [comment]
@Random-Liu please review.

File: jobs/ci-kubernetes-e2e-cri-gce.env [hunk 1]
```diff
@@ -1,14 +1,7 @@
 ### job-env
 KUBELET_TEST_ARGS=--experimental-cri=true
 KUBE_FEATURE_GATES=StreamingProxyRedirects=true
-
-# This list should match the list in kubernetes-pull-build-test-e2e-gce.
```
Comment (Random-Liu) [inline jobs/ci-kubernetes-e2e-cri-gce.env]: keep this line?

File: jobs/pull-kubernetes-e2e-gce-cri.sh [hunk 2]
```diff
@@ -47,12 +47,9 @@ export GINKGO_TOLERATE_FLAKES="y"
 
 export E2E_NAME="cri-e2e-${NODE_NAME}-${EXECUTOR_NUMBER:-0}"
 export GINKGO_PARALLEL="y"
-# Temporarily skip the test "Network should set TCP CLOSE_WAIT timeout"
-# and "[k8s.io] Loadbalancing: L7 [k8s.io] Nginx should conform to Ingress spec"
-# because it relies on host port, which is not implemented in CRI integration yet.
-# TODO(random-liu): Re-enable the test.
 # This list should match the list in kubernetes-e2e-gce.
-export GINKGO_TEST_ARGS='--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|Network\sshould\sset\sTCP\sCLOSE_WAIT\stimeout|Nginx\sshould\sconform\sto\sIngress\sspec'
+export
```
Comment (Random-Liu) [inline jobs/pull-kubernetes-e2e-gce-cri.sh]: Why new line here?

File: jobs/ci-kubernetes-e2e-cri-gce.env (hunk 1 above)
Comment (yujuhong) [inline jobs/ci-kubernetes-e2e-cri-gce.env]: sure.

File: jobs/pull-kubernetes-e2e-gce-cri.sh (hunk 2 above)
Comment (yujuhong) [inline jobs/pull-kubernetes-e2e-gce-cri.sh]: I blame my editor. Fixed.

**Comment by yujuhong** [comment]
@Random-Liu merge this?

**Comment by Random-Liu** [comment]
@yujuhong Sorry, I forgot this is manually merged.

**Pull request merged**